lte/cloud/go/services/subscriberdb/storage/digest_lookup.go

```diff
@@ -0,0 +1,207 @@
+/*
+ Copyright 2020 The Magma Authors.
+
+ This source code is licensed under the BSD-style license found in the
+ LICENSE file in the root directory of this source tree.
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package storage
+
+import (
+	"database/sql"
+
+	"magma/orc8r/cloud/go/clock"
+	"magma/orc8r/cloud/go/sqorc"
+
+	"github.com/Masterminds/squirrel"
+	"github.com/pkg/errors"
+)
+
+type digestLookup struct {
+	db      *sql.DB
+	builder sqorc.StatementBuilder
+}
+
+type DigestInfo struct {
```

Comment:
This struct's fields are all private, so I'm assuming this struct itself is meant to be private as well. So call it `type digestInfo struct`